src/azul/__init__.py

```diff
@@ -511,27 +513,24 @@ def is_internal(self):
             #        https://github.com/DataBiosphere/azul/issues/2865
             return self.is_integration_test_catalog or self.name == 'dcp3'
 
+        def to_dict(self):
+            return {'name': self.name, 'atlas': self.atlas, 'plugins': self.plugins}
+
     @cached_property
     def catalogs(self) -> Mapping[CatalogName, Catalog]:
         """
         A mapping from catalog name to a mapping from plugin type to plugin
         package name.
         """
-        catalogs = os.environ['AZUL_CATALOGS'].split(',')
-        catalogs = {
-            catalog: self.Catalog(
-                name=catalog,
-                atlas=atlas,
-                plugins={
-                    plugin_type: plugin
-                    for plugin_type, plugin in map(splitter('/'), plugins)
-                }
-            )
-            for atlas, catalog, *plugins in map(splitter(':'), catalogs)
-        }
+        catalogs = {}
+        # Import locally to avoid conflicts with `azul.json` module during
+        # unit tests.
+        import json
+        for catalog_spec in json.loads(os.environ['AZUL_CATALOGS']):
+            c = self.Catalog(**catalog_spec)
```

Comment:
@hannes-ucsc 

Yes you did, however the usage is slightly awkward. 
```
CBSE:azul amar$ echo $AZUL_CATALOGS
[{"name": "dcp2", "atlas": "hca", "plugins": {"repository": "tdr", "metadata": "hca"}}, {"name": "dcp2ebi", "atlas": "hca", "plugins": {"repository": "tdr", "metadata": "hca"}}, {"name": "lungmap", "atlas": "lungmap", "plugins": {"repository": "tdr", "metadata": "hca"}}, {"name": "it2", "atlas": "hca", "plugins": {"repository": "tdr", "metadata": "hca"}}, {"name": "it2ebi", "atlas": "hca", "plugins": {"repository": "tdr", "metadata": "hca"}}, {"name": "it3lungmap", "atlas": "lungmap", "plugins": {"repository": "tdr", "metadata": "hca"}}]
```

The original format of the AZUL_CATALOGS was  a string with `{spec},{spec},...` and these were split on the `,`. We cant do the split with the `,` because of the usage of `,` within the JSON object that defines the Catalog.

Because of this the Config class has to load an array that contains the JSON Catalogs objects and split the Catalogs within the array, at which point the JSON catalog objects have been converted into a dictionary and we don't need to use Catalog.from_json....

Reply:
I wasn't proposing a semantic change, just a refactoring.